maps/MrPokemonsHouse.asm

```diff
@@ -28,8 +28,7 @@ MrPokemonsHouse_MapScripts:
 	writetext MrPokemonIntroText2
 	promptbutton
 	waitsfx
-	giveitem MYSTERY_EGG
-	writetext MrPokemonsHouse_GotEggText
```

Comment:
For now I suggest keeping this in case its decided that this change should be locked behind a flag. I suspect that it probably won't need to be though, so once this gets properly merged into the main branch for speedchoice V7 we should propose to delete this.

Reply:
same as the sudowoodo code above, then; i don't think this code should be deleted.